composer.json

```diff
@@ -47,5 +47,9 @@
     },
     "require": {
         "php": "~7.1"
-    }
+    },
+    "scripts": {
+        "tests": "phpunit --exclude-group performance",
```

Comment:
Do we actually have performance tests in here?

Reply:
Looks like not! Copied it from travis.